client/ayon_core/plugins/publish/collect_udim.py

```diff
@@ -0,0 +1,41 @@
+"""Publish UDIM tiles."""
+import re
+
+import pyblish.api
+from ayon_core.lib import BoolDef
+from ayon_core.pipeline import publish
+
+UDIM_REGEX = re.compile(r"(.*)\.(?P<udim>\d{4})\.(.*)")
```

Comment:
Allow either `.` or `_` separator to the udim numbers.
```suggestion
UDIM_REGEX = re.compile(r"(.*)[._](?P<udim>\d{4})\.(.*)")
```